nixos/modules/programs/direnv.nix

```diff
@@ -0,0 +1,111 @@
+{
+  lib,
+  config,
+  pkgs,
+  ...
+}: let
+  cfg = config.programs.direnv;
+in {
+  options.programs.direnv = {
+
+    enable = lib.mkEnableOption (lib.mdDoc ''
+      Whether to enable direnv integration. Takes care of both installation and
```

Comment:
mkEnableOption adds `Whether to enable ` to the beginning of your string for you.